internal/station5/src/test/java/Test5.java

```diff
@@ -35,7 +35,8 @@ public class Test5 {
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         System.setOut(new PrintStream(baos));
         Station5.main(null);
-        String expected = "エラーが発生しました\n処理が完了しました\n";
+        String br = System.getProperty("line.separator");
```

Comment:
```suggestion
        String br = System.getProperty('line.separator');
```
[imo]
Javaのことは知らないことも多いので、考慮済みだったら申し訳ないんですがダブルクオートで囲まれていると「%s」などがないか、文字列の中身まで見て判断をしにいくことにより認知負荷が増えるので、ここの指定は文字列リテラルとして取り扱われるシングルクオートでも良いのでは？と思いました！

Reply:
Javaはシングルクオート使わないんす・・・・

Reply:
コメントありがとうございます！

Java の場合、シングルクォーテーション（'）は単一の文字（char型）を表すために使用するので、コンパイルエラー出るかも？みたいなのと、入門編のテストコードがダブルクオートだったので一旦揃えようかなと思ってます！

もしくはこれがいいみたい？
String br = System.lineSeparator();

Reply:
そうなんだ・・・！w
↑であれば、計算が出してるメソッドを利用した方が補完が効くので良さそうですね！

Reply:
> String br = System.lineSeparator();

こちらを推奨します！
別のStationではgetPropetyをつかってるかも・・・だけど、そちらもいずれ治す想定で。

Reply:
ありがとうございます！
修正しました！